spec/latest/core/index.html

```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
       <i>ApiDocumentation</i>.</p>
 
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra allows also to document both <i>ApiDocumentation</i> and
+      hypermedia controls with human-readable descriptions by applying
+      <i>hydra:title</i> and <i>hydra:description</i> (as shown in the
+      previous examples above). The former states a name of such a
+      decorated element that could be displayed to a human, the latter
+      provides it's description to be presented to that human i.e. as a hint.</p>
+
+    <p>Mentioned <i>hydra:title</i> and <i>hydra:description</i> SHOULD
+      override standard <i>rdfs:label</i> and <i>rdfs:comment</i> and presented
```

Comment:
It's not really overriding anything. 

```suggestion
      take precedence over standard <i>rdfs:label</i> and <i>rdfs:comment</i> and presented
```

I also question the word `standard` but it's not really incorrect.

Reply:
Maybe `common` would fit better?

Reply:
No strong opinion here. We could keep it as is

Reply:
Than let's leave it as it is now.